Sources/ApodiniNetworkingHTTPSupport/Headers.swift

```diff
@@ -599,3 +599,37 @@ extension Array: HTTPHeaderFieldValueCodable where Element: HTTPHeaderFieldValue
             .joined(separator: ", ")
     }
 }
+
+public extension AnyHTTPHeaderName {
+    /// The `Access-Control-Allow-Origin` header field
```

Comment:
Can you move this to the `extension` we already have somewhere above that contains all of the other non-HTTP2-specific header names?

Reply:
Yes good catch 💪